proxy-node-gateway/src/main/java/uk/gov/ida/notification/resources/EidasAuthnRequestResourceV2.java

```diff
@@ -34,16 +34,16 @@
     public static final String SESSION_KEY_HUB_REQUEST_ID = "hub_request_id";
     public static final String SUBMIT_BUTTON_TEXT = "Post Verify Authn Request to Hub";
 
-    private final EidasSamlParserService eidasSamlParserService;
-    private final VSPService vspService;
+    private final EidasSamlParserProxy eidasSamlParserService;
+    private final VerifyServiceProviderProxy vspProxy;
     private final SamlFormViewBuilder samlFormViewBuilder;
 
     public EidasAuthnRequestResourceV2(
```

Comment:
Can we remove the V2 later on when everything is working and delete the old version?

Reply:
Phil's got a PR which has just been approved which is removing the V2 and just having one resource. We're gonna get this in first then rebase his on top.